wbooks/src/app/components/CustomText/index.tsx

```diff
@@ -9,7 +9,11 @@ interface Props {
 }
 
 function CustomText({ text, style }: Props) {
-  return <Text style={[styles.text, style]}>{text}</Text>;
+  return (
+    <Text style={[styles.text, style]} numberOfLines={1}>
```

Comment:
No, you must only use  this prop in the Book title text, not in all the CustomText ⚠️